src/Fixer/Phpdoc/AlignMultilineCommentFixer.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Phpdoc;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ */
+final class AlignMultilineCommentFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
+{
+    private $tokenKinds;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $this->tokenKinds = [T_DOC_COMMENT];
+        if ($this->configuration['multiline_comment_blocks']) {
+            $this->tokenKinds[] = T_COMMENT;
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Multiline doc comment: enforce asterisk start on each line [PSR-5] and align them.',
+            [
+                new CodeSample(
+'<?php
+    /**
+            * This is a DOC Comment
+with a line not prefixed with asterisk
+
+   */'
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+*/',
+                    ['multiline_comment_blocks' => true]
+                ),
+                new CodeSample(
+'<?php
+    /*
+            * This is a doc-like multiline comment
+with a line not prefixed with asterisk
+
+   */',
+                    [
+                        'multiline_comment_blocks' => true,
+                        'multiline_comment_blocks_mixed_content' => true,
+                    ]
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isAnyTokenKindsFound([T_COMMENT, T_DOC_COMMENT]);
```

Comment:
actually, we do this. isCandidate is bounded to given configuration.
we use configuration inside `isCandidate` in multiple fixers. please do it here as well.

we call `isCandidate` directly in `fix` method:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/AbstractFixer.php#L64

for custom rules, we call it in runner:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Runner/Runner.php#L179

with custom runner, it's responsibility of dev to call it properly.

-------

Original issue was that one was able to call `fix` without calling `isCandidate/supports`. now, fix call them out of the box for our fixers and still require a manual run **before** fixing/**after** configuring for custom ones.
I do agree that if called before configuring only, it may provide wrong output. 
`FixerInterface::isCandidate` and `FixerInterface::supports` are implementation details that shall not be part of interface (yet, part of `AbstractFixer`)